webapp/src/Entity/Contest.php

```diff
@@ -1211,29 +1211,29 @@ public function validate(ExecutionContextInterface $context)
             }
         }
 
-        if ($this->processAwards) {
-            if ($this->goldAwards === null) {
+        if ($this->medalsEnabled) {
+            if ($this->goldMedals === null) {
                 $context
-                    ->buildViolation('This field is required when \'Process awards\' is set.')
-                    ->atPath('goldAwards')
+                    ->buildViolation('This field is required when \'Process medals\' is set.')
```

Comment:
"Enable medals". Is repeated 3 times below so maybe an indication the code could be a bit less repetitive, at least define the string only once?